src/main/kotlin/com/trivigenai/genai/GenAI.kt

```diff
@@ -0,0 +1,83 @@
+package com.trivigenai.genai
```

Comment:
From generateTrivia() can be extraced function that check if the returned category is good. It will improve code readability.